data/DEDIS/projects.yaml

```diff
@@ -531,3 +531,37 @@ projects:
     license: other
     date_added: 2022-03-16
     date_updated: 2022-03-16
+
+  apir:
+    name: APIR
+    categories:
+      - Privacy
+    applications:
+      - Info
+    description: >
```

Comment:
you can even put this as the `layman_description`, but then make sure that the description is still visible on showcase.c4dt.org